XRTK.Oculus/Packages/com.xrtk.oculus/Runtime/OculusPlatform.cs

```diff
@@ -16,11 +18,11 @@ public override bool IsBuildTargetAvailable
                 return (UnityEditor.EditorUserBuildSettings.activeBuildTarget == UnityEditor.BuildTarget.Android ||
                         UnityEditor.EditorUserBuildSettings.activeBuildTarget == UnityEditor.BuildTarget.StandaloneWindows ||
                         UnityEditor.EditorUserBuildSettings.activeBuildTarget == UnityEditor.BuildTarget.StandaloneWindows64) &&
-                       OculusApi.Version != null && OculusApi.Initialized;
```

Comment:
If we don't check if it's initialized will there be more than one data provider running at the same time?

While we do want the ability to run multiple of the same data providers, we don't in this case because they're both for the main camera rig.

Reply:
This could cause both the default camera provider and the oculus one to run at the same time, which we don't want.

Reply:
Well, there is no way to determine if the Oculus platform is running when you only have the editor open, as the headset is not running.
When you hit play in the editor, the IsAvailable property takes over.  Oculus doesn't have an "attached" API.  If you feel this is needed, comment further and I'll dig some more.

Reply:
I think `OculusApi.Initialized` comes back false all the time in the editor outside of playmode, is this correct?

Reply:
That is correct, Initialised only returns true when a headset is running.

Reply:
Yeah then I don't think this will give a proper result I think we're looking for. I'm gonna go ahead and merge this into that branch I'm working on to test it out